R/linear_pool.R

```diff
@@ -9,6 +9,15 @@
 #' @param ... parameters that are passed to `distfromq::make_q_fn`, specifying
 #'   details of how to estimate a quantile function from provided quantile levels
 #'   and quantile values for `output_type` `"quantile"`.
+#' @param compound_taskid_set `character` vector of the compound task ID variable
+#'   set. This argument is only relevant for `output_type` `"sample"`. NULL means
+#'   that samples are from a multivariate joint distribution across all levels of
+#'   all task id variables, while equality to `task_id_cols` means that the samples
+#'   are from separate univariate distributions for each individual prediction task.
+#'   NA means the compound_taskid_set is not relevant for the current modeling task.
+#'   Defaults to NA. Derived task ids must be included if ANY of the task ids their
```

Comment:
Changing "ANY" to "all" below:
```suggestion
#'   Defaults to NA. Derived task ids must be included if all of the task ids their
```
Here are some examples for the reference_date/horizon/target_date trio, regarding horizon as derived:
 - hub collects independent predictions for each horizon/target_date: reference_date and target_date go into the compound_taskid_set, so horizon must also go there
 - hub collects trajectories across horizon/target_date for each referene_date: compound_taskid_set includes reference_date, but not horizon or target_date